python/lsst/meas/astrom/match_probabilistic_task.py

```diff
@@ -124,20 +129,28 @@ def match(
 
         Returns
         -------
-        catalog_out_ref : `pandas.DataFrame`
+        catalog_out_ref : `pandas.DataFrame` | `astropy.table.Table`
             Reference matched catalog with indices of target matches.
-        catalog_out_target : `pandas.DataFrame`
+        catalog_out_target : `pandas.DataFrame` | `astropy.table.Table`
             Reference matched catalog with indices of target matches.
         """
         if logger is None:
             logger = self.log
 
+        is_ref_pd = isinstance(catalog_ref, pd.DataFrame)
+        is_target_pd = isinstance(catalog_target, pd.DataFrame)
+        if is_ref_pd or is_target_pd:
+            logger.warning("pandas DataFrame inputs are deprecated")
```

Comment:
Is there a deprecation ticket?  There needs to be a ticket, and the warning should use a python deprecation warning, probably following https://developer.lsst.io/stack/deprecating-interfaces.html#package-deprecation if we can't use the sphinx stuff on a call.